src/bp-members/bp-members-adminbar.php

```diff
@@ -165,106 +164,182 @@ function bp_members_admin_bar_user_admin_menu() {
 add_action( 'admin_bar_menu', 'bp_members_admin_bar_user_admin_menu', 99 );
 
 /**
- * Build the "Notifications" dropdown.
+ * Build the Admin or Members "Notifications" dropdown.
+ *
+ * @since 1.5.0
  *
- * @since 11.4.0
+ * @return bool
  */
-function bp_members_admin_bar_notifications_dropdown( $notifications = array(), $menu_link = '', $type = 'members' ) {
-	if ( ! $menu_link || ( 'admin' === $type && empty( $notifications ) ) ) {
-		return false;
+function bp_members_admin_bar_notifications_menu() {
+	if ( ! is_user_logged_in() ) {
+		return;
 	}
 
 	global $wp_admin_bar;
 
-	$count       = 0;
-	$alert_class = array( 'count', 'no-alert' );
+	// Init some variables.
+	$user_id             = bp_loggedin_user_id();
+	$count               = 0;
+	$notices_count       = 0;
+	$notices             = array();
+	$notifications_count = 0;
+	$all_items_link      = '';
+	$alert_class         = array( 'count', 'no-alert' );
+
+	if ( bp_is_active( 'members', 'notices' ) ) {
+		$all_items_link = bp_get_member_all_notices_url();
+		$notices_count  = bp_members_get_notices_count(
+			array(
+				'user_id'  => $user_id,
+				'exclude'  => bp_members_get_dismissed_notices_for_user( $user_id ),
+			)
+		);
+
+		if ( $notices_count ) {
+			$alert_class = array( 'pending-count', 'alert', 'admin-type' );
+			$notices     = array(
+				(object) array(
+					'id'      => 'notices',
+					'content' => sprintf(
+						/* translators: %s: total notices count */
+						_n( 'Warning: you have %s unread notice', 'Warning: you have %s unread notices', $notices_count, 'buddypress' ), number_format_i18n( $notices_count )
```

Comment:
"Warning" is a little too strong of a word. How about "Important: You have %s unread notice"? (The American English style is to often capitalize the first word after a colon if the phrase is a sentence.)